src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -56,6 +63,26 @@ private void setStyleToDefault() {
         commandTextField.getStyleClass().remove(ERROR_STYLE_CLASS);
     }
 
+    private void autoComplete(KeyEvent e) {
+        if (e.getCode() != KeyCode.TAB) {
+            return;
+        }
+
+        // Prevent the default Tab behavior
+        e.consume();
```

Comment:
Maybe we can apply SLAP here and have another method `handleAutoComplete` that handles autocompletion, moving the lines after this into that method, and just have this method for checking the key code.
```suggestion
        if (e.getCode() == KeyCode.TAB) {
            handleAutoComplete();
            // Prevent the default Tab behavior
            e.consume();
        }
```